Jellyfin.Plugin.TMDbBoxSets/TMDbBoxSetManager.cs

```diff
@@ -98,13 +105,32 @@ private IReadOnlyCollection<BoxSet> GetAllBoxSetsFromLibrary()
         {
             return _libraryManager.GetItemList(new InternalItemsQuery
             {
-                IncludeItemTypes = new[] {nameof(BoxSet)},
+                IncludeItemTypes = new[] { nameof(BoxSet) },
                 CollapseBoxSetItems = false,
                 Recursive = true,
                 HasTmdbId = true
             }).Select(b => b as BoxSet).ToList();
         }
 
+        private string GetTmdbCollectionName(IReadOnlyCollection<Movie> movies)
+        {
+            var collectionNames = movies.Select(movie => movie.TmdbCollectionName).Where(name => !string.IsNullOrWhiteSpace(name));
```

Comment:
`collectionNames` is being enumerated multiple times, either call `.ToList()` or refactor the function

Reply:
Done. Thanks for the review. 😉